packs/github/actions/add_team_membership.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name: add_team_membership
+runner_type: run-python
+description: Add (and invite if not a member) a user to a team
+enabled: true
+entry_point: add_team_membership.py
+parameters:
+  user:
+    type: "string"
+    description: "User to invite"
+    required: true
+  organization:
+    type: "string"
+    description: "Organization the team belongs to"
+    required: true
+  team:
```

Comment:
Probably better to be explicit and call it `team_id`, but would need to check existing actions. Maybe they don't use `id` suffix and current approach is better for consistency,